exercises/triangle/triangle_test.py

```diff
@@ -1,67 +1,65 @@
 import unittest
 
-from triangle import Triangle, TriangleError
+from triangle import equilateral, isosceles, scalene
 
 
-class TriangleTests(unittest.TestCase):
-    def test_equilateral_triangles_have_equal_sides(self):
-        self.assertEqual(Triangle(2, 2, 2).kind(), "equilateral")
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
 
-    def test_larger_equilateral_triangles_also_have_equal_sides(self):
-        self.assertEqual(Triangle(10, 10, 10).kind(), "equilateral")
+class EquilateralTests(unittest.TestCase):
+    def test_true_if_all_sides_are_equal(self):
+        self.assertIs(equilateral([2, 2, 2]), True)
 
-    def test_isosceles_triangles_have_last_two_sides_equal(self):
-        self.assertEqual(Triangle(3, 4, 4).kind(), "isosceles")
+    def test_false_if_any_side_is_unequal(self):
+        self.assertIs(equilateral([2, 3, 2]), False)
 
-    def test_isosceles_triangles_have_first_and_last_sides_equal(self):
-        self.assertEqual(Triangle(4, 3, 4).kind(), "isosceles")
+    def test_false_if_no_sides_are_equal(self):
+        self.assertIs(equilateral([5, 4, 6]), False)
 
-    def test_isosceles_triangles_have_two_first_sides_equal(self):
-        self.assertEqual(Triangle(4, 4, 3).kind(), "isosceles")
+    def test_false_if_all_sides_are_zero(self):
+        self.assertIs(equilateral([0, 0, 0]), False)
```

Comment:
I feel like this should be raising an exception, rather than just returning `False` - I know this is in the canonical data, but we're asking whether a point is an equilateral triangle and just ignoring the fact that they are different things.

Reply:
I understand what you're saying, and under the old format of `kind() = 'equilateral'|'isosceles'|'scalene'`, I would've preferred raising an exception, because the usage of `kind()` assumed that the input was indeed a triangle. However, the question of "Is a point an equilateral triangle?" is a valid one, even if it isn't one that would normally need asking.